lib/tasks/instrumentation_generator/templates/chain.tt

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic::Agent::Instrumentation
+  module <%= @class_name %>
+    def self.instrument!
+      ::<%= @class_name %>.class_eval do
+        include NewRelic::Agent::Instrumentation::<%= @class_name %>
+
+        alias_method :<%= @method.downcase %>_without_new_relic, :<%= @method.downcase %>
+
+        def <%= @method.downcase %><%= "(#{@args})" if @args %>
+          <%= @method.downcase %>_with_tracing<%= "(#{@args})" if @args %> do
+            <%= @method.downcase %>_without_new_relic<%= "(#{@args})" if @args %>
```

Comment:
@fallwith  - Here's an example of where `@args` is used in a template to pass the arguments.